lesson09/script.js

```diff
@@ -25,6 +25,7 @@ function showCategories(result){
 }
 
 function SetCategoriesAfter3Seconds(){
+    showLoadingImg();
     return new Promise(resolve => {
     const categories = [
         {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
```

Comment:
I changed the function showLoadingImg () to the proper position.
Because "showLoadingImg()" runs for 3 seconds before the Promise runs.